source/extensions/stat_sinks/common/statsd/statsd.cc

```diff
@@ -124,8 +125,8 @@ void TcpStatsdSink::TlsSink::beginFlush(bool expect_empty_buffer) {
 
 void TcpStatsdSink::TlsSink::commonFlush(const std::string& name, uint64_t value, char stat_type) {
   ASSERT(current_slice_mem_ != nullptr);
-  // 40 > 6 (prefix) + 4 (random chars) + 30 for number (bigger than it will ever be)
-  const uint32_t max_size = name.size() + 40;
+  // 34 >  4 (random chars) + 30 for number (bigger than it will ever be)
```

Comment:
Can you update this comment to:
`34 > 4 (postfix chars, e.g., "|ms\n") + 30 for number (bigger than it will ever be)`

Reply:
Done!